### PR TITLE
update plugin

### PR DESCRIPTION
**Pull request by azazellochg**
- add 0.2.3 support
-func def are now similar to cryolo plugin
- add denoise step

@pconesa I don't know who can review this.. :)

### CODE REVIEW

**Comment by azazellochg** [comment]
wrong branch

**Comment by pconesa** [comment]
We can! Maybe @JorMaister  that is maintaining sphire and is similar.

File: README.rst [hunk 1]
```diff
@@ -47,15 +47,15 @@ The following steps presuppose that you have Anaconda or Miniconda installed on
   This is needed to activate the conda environment.
 | For further information please visit the following website:
 | https://github.com/conda/conda/blob/master/CHANGELOG.md#440-2017-12-20
-| Set TOPAZ_ACTIVATION_CMD variable in the Packages section. 
-| For example: ``TOPAZ_ACTIVATION_CMD = conda activate topaz``
-| This will activate the conda environment with the default name topaz.
+| Set TOPAZ_ENV_ACTIVATION variable in the Packages section.
```
Comment (pconesa) [inline README.rst:50]: I think this is optional....should we mention that?

File: setup.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
     # For a discussion on single-sourcing the version across setup.py and the
     # project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.1',  # Required
+    version='1.2',  # Required
```
Comment (pconesa) [inline setup.py:43]: We decided to bump version to 3.0.

File: README.rst (hunk 1 above)
Comment (azazellochg) [reply README.rst:50]: True, I just didn't want to change much prévious code. Ps. You are looking at the closed pr.

**Pull request closed without merging**